dist/tools/kconfiglib/riot_kconfig.py

```diff
@@ -0,0 +1,54 @@
+""" RIOT customization of Kconfig """
+import sys
+
+from kconfiglib import Kconfig, KconfigError
+
+
+class RiotKconfig(Kconfig):
+    """ RIOT adaption of Kconfig class """
+    def _parse_help(self, node):
+        """ Parses the help section of a node, removing Doxygen markers """
+        DOXYGEN_MARKERS = ["@ref ", "@see "]
+
+        # call default parsing
+        super(RiotKconfig, self)._parse_help(node)
+
+        # remove Doxygen markers
+        for marker in DOXYGEN_MARKERS:
+            node.help = node.help.replace(marker, "")
+
+
+def standard_riot_kconfig(description=None):
+    """
+    Argument parsing helper for tools that take a single optional Kconfig file
+    argument (default: Kconfig). Returns the RiotKconfig instance for the parsed
+    configuration. Uses argparse internally.
+
+    Exits with sys.exit() (which raises SystemExit) on errors.
+
+    description (default: None):
+      The 'description' passed to argparse.ArgumentParser().
+      argparse.RawDescriptionHelpFormatter is used, so formatting is preserved.
+    """
+    import argparse
```

Comment:
Should be added at the beginning of the module

Reply:
Moved